packages/lesmis-server/examples.http

```diff
@@ -0,0 +1,30 @@
+@hostname = localhost
+@port = 8060
+@host = {{hostname}}:{{port}}
+@version = v1
+@contentType = application/json
+
+### Test
+GET http://{{host}}/{{version}}/test HTTP/1.1
+content-type: {{contentType}}
+
+###
+
+# @name login
+POST http://{{host}}/{{version}}/login HTTP/1.1
+content-type: {{contentType}}
+
+{
+    "emailAddress": "{{email}}",
+    "password": "{{password}}"
+}
+
+###
+
+@sessionCookie = {{login.response.headers.Set-Cookie}}
```

Comment:
This is cool, it allows u to use the `login` request to get the session cookie and the pass it on in other requests 👍